code/modules/mob/living/carbon/human/species.dm

```diff
@@ -432,6 +432,12 @@ GLOBAL_LIST_EMPTY(features_by_species)
 		var/obj/item/thing = C.get_item_by_slot(slot_id)
 		if(thing && (!thing.species_exception || !is_type_in_list(src,thing.species_exception)))
 			C.dropItemToGround(thing)
+	if(ishuman(C))
```

Comment:
pointless check. The below proc is carbon-level and will automatically call the human-level version when invoked.